python/paddle/fluid/tests/performance_tuning/DeepASR/run_on_1card.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+export PYTHONUNBUFFERED=1 
+export CUDA_VISIBLE_DEVICES=7 
```

Comment:
Is the magic number 7 due to that the computer you are using has 8 GPU cards? Is it possible to make it less "magical" by, say, calling nvidia-smi or some other commands to retrieve the number of GPUs on the system?